docs/2_6_versioning_layered_standards.adoc

```diff
@@ -66,3 +66,12 @@ A minimal example of a manifest file, for layered standards that have no need fo
 ----
 include::examples/fmi_ls_manifest_minimal_example.xml[]
 ----
+
+The standard includes a schema for the attribute definitions and the minimal example element as `fmi3LSManifest.xsd`.
```

Comment:
What is the "minimal example element"? The default root element `FMILSManifest`?

Reply:
Adding a diagram would not help much, since it would only show one box given our diagram capabilities. Additionally the schema is for other schema authors, hence they should be capable of reading or viewing it in their own tools...